src/woo_search/search_index/api/serializers/search.py

```diff
@@ -39,6 +40,93 @@ class SearchSerializer(serializers.Serializer):
         default=ResultTypeChoices.all,
         help_text=_("Which table(s) should be present in the returning body."),
     )
+    registratiedatum_vanaf = serializers.DateTimeField(
+        required=False,
+        allow_null=True,
+        default=None,
+        help_text=_("Filter results registered after or on the given value."),
+    )
+    registratiedatum_tot = serializers.DateTimeField(
+        required=False,
+        allow_null=True,
+        default=None,
+        help_text=_("Filter results registered before or on the given value."),
+    )
```

Comment:
if it also includes results on the given value, `tot en met` would be more correct (wouldn't make the query param name prettier though :sweat_smile:)

Reply:
I've opted to stick to the original [specified schema](https://github.com/GPP-Woo/GPP-zoeken/issues/13#issuecomment-2647992308) from which the wording implies `lt` rather than `lte`. The `creatiedatum` filter has `lte` behaviour, and the parameter name was updated accordingly (this field was not originally in the schema linked above, it's a separate requirement in the user story).

Adapting it in either direction is easy, so let us know if it should be changed!